examples/kubernetes/connectivity-check/connectivity-check-ipv6.yaml

```diff
@@ -0,0 +1,1338 @@
+# Automatically generated by Makefile. DO NOT EDIT
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo-a
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+  selector:
+    name: echo-a
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo-a
+spec:
+  selector:
+    matchLabels:
+      name: echo-a
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: echo-a
+    spec:
+      containers:
+        - name: echo-container
+          image: docker.io/cilium/json-mock:1.2
+          imagePullPolicy: IfNotPresent
+          readinessProbe:
+            exec:
+              command: ["curl", "-sS", "--fail", "-o", "/dev/null", "localhost"]
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo-b
+spec:
+  type: NodePort
+  ports:
+    - port: 80
+      nodePort: 31313
+  selector:
+    name: echo-b
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo-b-headless
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - port: 80
+  selector:
+    name: echo-b
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo-b
+spec:
+  selector:
+    matchLabels:
+      name: echo-b
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: echo-b
+    spec:
+      containers:
+        - name: echo-container
+          image: docker.io/cilium/json-mock:1.2
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 80
+              hostPort: 40000
+          readinessProbe:
+            exec:
+              command: ["curl", "-sS", "--fail", "-o", "/dev/null", "localhost"]
+---
+# The echo-b-host pod runs in host networking on the same node as echo-b.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo-b-host
+spec:
+  selector:
+    matchLabels:
+      name: echo-b-host
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: echo-b-host
+    spec:
+      hostNetwork: true
+      containers:
+        - name: echo-container
+          image: docker.io/cilium/json-mock:1.2
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: PORT
+              value: "41000"
+          readinessProbe:
+            exec:
+              command: ["curl", "-sS", "--fail", "-o", "/dev/null", "localhost:41000"]
+      affinity:
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: name
+                    operator: In
+                    values:
+                      - echo-b
+              topologyKey: "kubernetes.io/hostname"
+---
+# Connecting to "echo-b-host-headless" will provide service discovery to the
+# node IP on which echo-b* is running
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo-b-host-headless
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    name: echo-b-host
+---
+apiVersion: v1
+kind: Service
```

Comment:
Once https://github.com/cilium/cilium/issues/9776 has been fixed, we can set the `ipFamily`.

Reply:
It does not hurt to set it before #9776 is fixed.

Reply:
sure, I tried to achieve this based on flag ipFamly  from cue command, but the code is quite ugly, maybe I need to read cuelang docs a little bit more. Let me do it as follow up.